src/jdk.management.agent/share/conf/management.properties

```diff
@@ -254,8 +254,8 @@
 
 # ################ Management agent listen interface #########################
 #
-# com.sun.management.jmxremote.host=<host-or-interface-name>
-#      Specifies the local interface on which the JMX RMI agent will bind.
+# com.sun.management.jmxremote.host=<host-name-or-address>
+#      Specifies the address on which the JMX RMI agent will bind.
```

Comment:
The placeholder is "host-name-or-address" so it might be clearer to say the local host name or IP address in the description.

Reply:
ok!